### DISCUSSION TITLE
Malware?

### DISCUSSION THREAD

**Discussion by MESWEB** [MajkiIT/polish-ads-filter#17882] [General]
Mam ostatnio jakiś dziwny przypadek gdy przeglądam internet. Czasem gdy klikam jakieś linki wtedy pojawiają się dziwne linki które są blokowane przez uMatrix. Na załączonym screenie jest przykład po kliknięciu linka z wyników wyszukiwarki Google. Watpie aby Google zrobiło taki scam chyba ze ja cos przespałem?
Windows 10 
Opera
![Opera Snapshot_2020-11-27_211317_clblbeknmgobkgonndomehcjpckopfeh](https://user-images.githubusercontent.com/2931024/100485010-73ee0980-30f6-11eb-932a-18361f579fe1.png)
Co może być tego przyczyna?

**Comment by zaba-monika** [comment]
- wyszukiwarka Google jest pełna takich niebezpiecznych linków, to nie wina Google, Google tylko indeksuje strony i widać, niektóre strony jakoś przedostają się przez mechanizmy bezpieczeństwa Google, być może na początku takie strony były normalne, tylko po to żeby zostać zaindeksowanymi w wyszukiwarce Google, a potem, autor strony podmienił zawartość /  dodał malware, albo strona zhackowana
- był nawet wątek z takimi zgłaszanymi linkami z Google tutaj: https://github.com/PolishFiltersTeam/KAD/issues - 
nazywał się "Kilka fejkowych stron z fałyszywmi nagrodami/loteriami", ale komuś poleciały konta to i poleciał wątek...
- nie wiem czy wiesz ale, uMatrix nie jest już rozwijany: https://github.com/gorhill/uMatrix , a jego repozytorium zostało zarchiwizowane, ale zdaje się, że ktoś gdzieś zrobił forka i kontynuuje

**Reply by MESWEB** [reply]
Wątpię  aby strona viber.com mogła pozwolić sobie na takie zagrania. To tyczy się również innych dużych serwisów internetowych. Podejrzewam że może to być przeglądarka, dodatek lub crack jaki został użyty do Office 2016.
A jest jakaś alternatywa dla uMatrix?

**Reply by zaba-monika** [reply]
- strona, która została zablokowana to `click.linksynergy.com` a nie `viber.com` , wejdź bezpośrednio na `viber.com` - raczej ci nie zablokuje
- co do alternatywy dla `uMatrix` nie wiem, ale raczej nie ma, ale gdzieś chyba ktoś jakiś fork zrobił i kontynuuje tylko nie pamiętam gdzie, trzebaby poszukać

**Reply by hawkeye116477** [reply]
U mnie często tak się zdarzało, kiedy wchodziłem na stronę rozkładów MPK albo np na `grupamerlin.pl`. 

W drugim przypadku, jeżeli wejdziesz bezpośrednio, to nic się nie pojawia, a jak przez Google albo inną wyszukiwarkę, to od razu wygrałeś smartfona.

Nie wiem co sobie myśleć o tej Grupie Merlin, bo to niby jakaś agencja reklamowa... Ale jeżeli chodzi o MPK to strona była nawet oznaczona przez Google jako zhackowana, choć wcześniej wchodziłem bezpośrednio, więc nawet o tym nie wiedziałem.

**Reply by krystian3w** [reply]
Kontynuacja uMatrix do ręcznej kompilacji: https://github.com/geekprojects/nuTensor/

**Reply by krystian3w** [reply]
Co do źródła to może zakatowali polski opis strony: `https://www.viber.com/pl/` - jakiś błąd 404 i nie chce przeskoczyć na katalog `/en/`.

**Comment by MajkiIT** [comment]
@MESWEB przeskanuj kompa softem adwcleaner.

**Reply by zaba-monika** [reply]
Nom, w sumie możliwe, że to jakiś syf, ktoś też pisał, że zapomniał, że miał AdAway zainstalowany, i po odinstalowaniu ustąpiło.

**Reply by hawkeye116477** [reply]
Lektura do poczytania => https://malwaretips.com/blogs/remove-you-have-won-a-google-gift/

> zapomniał, że miał AdAway zainstalowany, i po odinstalowaniu ustąpiło.

AdAway to otwarto źródłowy adblocker. Raczej wątpię, by on to powodował, no chyba, że może był ściągnięty z jakiegoś podejrzanego źródła.

**Reply by zaba-monika** [reply]
Z tym AdAway to odnalazłam link do posta tego gościa:
https://forums.anandtech.com/threads/links-dont-display-redirects-to-click-linksynergy-solved.2496976/post-38681725

**Reply by MESWEB** [reply]
> Z tym AdAway to odnalazłam link do posta tego gościa:
> https://forums.anandtech.com/threads/links-dont-display-redirects-to-click-linksynergy-solved.2496976/post-38681725

Ja nic nie instalowalem w tym stylu. Jedynie skrypty Tampermonkey. Wylaczylem cda.pl przeskakiwanie reklam. Moze pomoze.

**Reply by krystian3w** [reply]
To jest infekcja na serwerze danej strony, drugiej wizyty już nie obsługują, bo IP leci do bazy by ktoś nie mógł ustalać czemu się otwierają iPhony bez folii czy samsungi wypadające z ciężarówki.

Co do wchodzenia bezpośrednio na zainfekowaną stronę z nowym IP to raczej na 99% i tak się odpali ten syf.

Serwera za bardzo nie wyleczysz filtrami HTML (kiedyś ten syf używał po prostu tagu `<meta http-equiv="refresh" content="0; url=https://example.com/">`)  ani skryptletami - poza tym trzeba by jakieś bezpieczne metody wciskać w każdą stronę WWW.

**Reply by MESWEB** [reply]
@krystian3w Wczoraj mi sie to pokazalo na allegro gdy chcialem kliknac link itemu. Czyli allegro by tez sie bawilo w zbieranie $0.01 za klik w link? Sadze ze nie pozwoliliby sobie na takie przekierowania.
